routes/web.php

```diff
@@ -36,9 +35,12 @@
 
     $ids = $request->input('ids');
 
+    $results = app('app\Http\Controllers\MismatchController')->index($request);
```

Comment:
:star_struck: NICE!! :star_struck:

Reply:
I am not sure about what could be good alternatives. Maybe @itamargiv or @mariushoch  could offer a suggestion?

Reply:
To clarify, I love this approach as it seems so easy, and I think it's a nice original, and pretty elegant solution that reuses already existing code. I must say, though, that I left this comment without doing a full review, so it is a bit out of context.

@Silvan-WMDE is of course right, the logic for the web endpoint and the logic for the API endpoint are not fully compatible. And we perhaps don't necessarily want to rely on the results of the API endpoint for this specific view of the data in the database. Therefore, the only solution I can think of is that you query the database in this route handler as well, especially since we still have some other operations to perform on the data before passing it to the page, such as fetching entity labels from the Wikidata API.

Reply:
Very good points! I've updated the PR following @Silvan-WMDE 's suggestion.